src/chocolatey.resources/helpers/functions/Install-ChocolateyInstallPackage.ps1

```diff
@@ -287,60 +293,77 @@ Pro / Business supports a single, ubiquitous install directory option.
   if ($env:ChocolateyInstall -and $ignoreFile -match [System.Text.RegularExpressions.Regex]::Escape($env:ChocolateyInstall)) {
     try {
       '' | out-file $ignoreFile
-    } catch {
+    }
+    catch {
       Write-Warning "Unable to generate `'$ignoreFile`'"
     }
   }
 
   $workingDirectory = Get-Location -PSProvider "FileSystem"
   try {
     $workingDirectory = [System.IO.Path]::GetDirectoryName($fileFullPath)
-  } catch {
+  }
+  catch {
     Write-Warning "Unable to set the working directory for installer to location of '$fileFullPath'"
     $workingDirectory = $env:TEMP
   }
 
   try {
     # make sure any logging folder exists
     $pattern = "(?:['`"])([a-zA-Z]\:\\[^'`"]+)(?:[`"'])|([a-zA-Z]\:\\[\S]+)"
-    $silentArgs, $additionalInstallArgs | %{ Select-String $pattern -input $_ -AllMatches } |
-      % { $_.Matches } | % {
-        $argDirectory = $_.Groups[1]
-        if ($argDirectory -eq $null -or $argDirectory -eq '') { continue }
-        $argDirectory = [System.IO.Path]::GetFullPath([System.IO.Path]::GetDirectoryName($argDirectory))
-        Write-Debug "Ensuring '$argDirectory' exists"
-        if (![System.IO.Directory]::Exists($argDirectory)) { [System.IO.Directory]::CreateDirectory($argDirectory) | Out-Null }
-      }
-  } catch {
+    $silentArgs, $additionalInstallArgs | % { Select-String $pattern -input $_ -AllMatches } |
+    % { $_.Matches } | % {
+      $argDirectory = $_.Groups[1]
+      if ($argDirectory -eq $null -or $argDirectory -eq '') { continue }
+      $argDirectory = [System.IO.Path]::GetFullPath([System.IO.Path]::GetDirectoryName($argDirectory))
+      Write-Debug "Ensuring '$argDirectory' exists"
+      if (![System.IO.Directory]::Exists($argDirectory)) { [System.IO.Directory]::CreateDirectory($argDirectory) | Out-Null }
+    }
+  }
+  catch {
     Write-Debug "Error ensuring directories exist -  $($_.Exception.Message)"
   }
 
   if ($fileType -like 'msi') {
-    $msiArgs = "/i `"$fileFullPath`""

```

Comment:
We seem to have lost the initial setting of the `$msiArgs` here, which I believe is still required.  Can you confirm?

Reply:
Hmm. Looking at this change... my read is that it should be something like this:

```powershell
$msiArgs = if ($overrideArguments) {
      Write-Host "Overriding package arguments with '$additonalInstallArgs' (replacing '$silentArgs')"
      "$additionalInstallArgs"
} else {
      "$silentArgs $additionalInstallArgs"
}

```

Reply:
Unless I am reading something wrong, the only place that we were setting the msiArgs to `/i `"$fileFullPath`"` has now been removed. I am not seeing this anywhere else, so I feel that this is a regression.  Happy to be proven wrong though.

Reply:
Hmm... no, you're absolutely right. Yeah, we need that portion pulled back in. That can either just be a part of this if/else or we can set it before this if/else and go from there. (Ignore my previous suggestion in that case).

Reply:
Corrected this regression.